packages/playground/src/weblets/profile_manager.vue

```diff
@@ -284,13 +287,93 @@ defineProps({
 });
 defineEmits<{ (event: "update:modelValue", value: boolean): void }>();
 
+let mountedTimeout: any;
+watch(
+  () => props.modelValue,
+  m => {
+    if (m) {
+      if (mountedTimeout) {
+        clearTimeout(mountedTimeout);
+      }
+      mountedTimeout = setTimeout(() => {
+        mounted();
+      });
+    }
+  },
+);
```

Comment:
Please don't use `setTimeout` api to wait as it might lead to unexpected results like old playground

Reply:
Used `requestAnimationFrame` instead

Reply:
Don't use `requestAnimationFrame` either.
Please check `nextTick`

```ts
import { nextTick } from "vue"

nextTicket().then(mounte)
```

Reply:
@Mahmoud-Emad any updates?

Reply:
should be done